Abcrete/BangazonCLI
# methods for payment type now complete with all tests passing

## Status
Choose:
**READY**

## Description
The methods to add payment type to customer and list the payment types for a customer are now complete. The corresponding tests pass too.

Also made a few changes to the Database interface file to be able to create an order table and also added a primary key and foreign keys to prodorder table

## Todos
- [* ] Tests
- [* ] Documentation

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout ajmethods
3. make sure the payment type tests pass
4. make sure order table is created and all other tables also create correctly

## Impacted Areas in Application
Anything connected with payment type and also the entire database and hence the entire application

## Link to Feature Ticket
#19, #22
